src/renderer/components/modal/swap/pages/done/BitcoinRefundedPage.tsx

```diff
@@ -1,38 +1,33 @@
 import { Box, DialogContentText } from '@material-ui/core';
 import { SwapStateBtcRefunded } from 'models/storeModel';
 import BitcoinTransactionInfoBox from '../../BitcoinTransactionInfoBox';
-import { useActiveDbState } from '../../../../../../store/hooks';
+import { useActiveSwapInfo } from '../../../../../../store/hooks';
 import FeedbackInfoBox from '../../../../pages/help/FeedbackInfoBox';
 
 export default function BitcoinRefundedPage({
   state,
 }: {
   state: SwapStateBtcRefunded | null;
 }) {
```

Comment:
**Suggestion:** Consider adding error handling or a fallback for when `swap` is null or undefined. This will prevent potential errors when trying to access `swap.btcRefundAddress`.
```suggestion
const swap = useActiveSwapInfo() || {};
const additionalContent = swap.btcRefundAddress
  ? `Refund address: ${swap.btcRefundAddress}`
  : 'Refund address: Not available';
```